package.json

```diff
@@ -9,7 +9,7 @@
     "dev:build": "rimraf dist/development && cross-env NODE_ENV=development webpack",
     "prod:build": "rimraf dist/production && cross-env NODE_ENV=production webpack -p",
     "lint": "eslint ./",
-    "test": "yarn lint",
```

Comment:
@Prerna1993 Could you explain why this command was removed from the test script?